### ISSUE TITLE
[torrent2database] --add-files-limit not working as desired

### ISSUE THREAD

**Issue by DyonR** [DyonR/magnetico2bitmagnet#3]
I believe that the --add-files-limit just takes the first 500 records from the file list. 
If there are, let's say 400 padding files and 600 actual files, but the first 400 files in the list are padding files, then it will only import 100 actual files.  
Besides that, there also must be an option to add padding files, in case someone wants that (I don't know why but eh...)  
What it should do is;  
Read file from b'files', check if it's padding, if it's padding, skip the file, if its not a padding, assign index 0 and go next, if not a padding again, index 1 and go next, if it's padding, skip it, if it's a file now, index 2 and go next.
Basically, I need to look at the `get_torrent_details` function, more specifically this:
```
        if file_status == "multi":
            files_count = len(info_dict[b'files'])
            for index, file in enumerate(info_dict[b'files'][:add_files_limit]):
                file_path = os.path.join(*[decode_with_fallback(part) for part in file[b'path']])
                file_size = file[b'length']
                total_size += file_size
                if "_____padding" not in file_path and ".____padding" not in file_path:
                    files_info.append((index, file_path, file_size))
```

**Comment by DyonR** [comment]
ChatGPT code that I can use as reference:   
```
if file_status == "multi":
    files_count = len(info_dict[b'files'])
    files_info = []  # To store info of non-padding files
    actual_files_count = 0  # Counter for non-padding files

    for file in info_dict[b'files']:
        file_path = os.path.join(*[decode_with_fallback(part) for part in file[b'path']])
        file_size = file[b'length']
        total_size += file_size

        # Check if the file is not padding, and we haven't reached the limit yet
        if ("_____padding" not in file_path and ".____padding" not in file_path) and actual_files_count < add_files_limit:
            files_info.append((actual_files_count, file_path, file_size))
            actual_files_count += 1  # Increment only for non-padding files

        # Exit loop early if we've reached the limit of non-padding files to add
        if actual_files_count >= add_files_limit:
            break
```
